subsys/bluetooth/audio/bap_broadcast_sink.c

```diff
@@ -417,7 +417,7 @@ static struct bt_bap_broadcast_sink *broadcast_sink_get_by_pa(struct bt_le_per_a
 
 static void broadcast_sink_add_src(struct bt_bap_broadcast_sink *sink)
 {
-	struct bt_bap_scan_delegator_add_src_param add_src_param;
+	struct bt_bap_scan_delegator_add_src_param add_src_param = {0};
```

Comment:
If you want to keep this change, you can remove `add_src_param.num_subgroups = 0U;`